shipyard/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3ea4d492-9d1c-4dda-bb8a-e571eca3ce7d",
+  "app_id": "shipyard",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "",
+    "title": "Shipyard",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "",
```

Comment:
Please provide a caption of the image

Reply:
Added captions for both images here: e24ab66684f3948876dfdebb39202df0136969df